syntax = "proto3";

package contracts.services.documents.contracts;

import "contracts/services/documents/contracts/contract.proto";

message CreateContractRequest {
  string account_id = 1;
  bytes content = 2;
}

message CreateContractResponse {
  Contract contract = 1;
}